source/Subsystem/Renderer/RenderContext.h

```diff
@@ -4,15 +4,17 @@
 
 class Camera;
 class ShaderLibrary;
+class TextureLibrary;
 
 class RenderContext
 {
 public:
 	RenderContext();
 	~RenderContext();
 
+	void Initialize();
 	void CreateBuffers(std::vector<Model>& aModels);
-	void Render(const std::vector<Model>& aModels, const ShaderLibrary& aShaderLibrary, int aWidth, int aHeight);
+	void Render(const std::vector<Model>& aModels, const TextureLibrary& aTextureLibrary, const ShaderLibrary& aShaderLibrary, int aWidth, int aHeight);
```

Comment:
Next update we might want to consider adding a ModelLibrary to fit the rest of the things.